packages/react-dom/src/client/ReactDOM.js

```diff
@@ -760,7 +761,11 @@ const DOMRenderer = ReactFiberReconciler({
     ): void {
       if (container.nodeType === COMMENT_NODE) {
         (container.parentNode: any).insertBefore(child, container);
+        trapClickOnNonInteractiveElement(
+          ((container.parentNode: any): HTMLElement),
+        );
```

Comment:
@gaearon is using a comment node as a mount point supported for portals as well?

Reply:
Worth supporting for consistency I guess